src/sys/local_offset_at/wasm_js.rs

```diff
@@ -0,0 +1,9 @@
+use crate::{OffsetDateTime, UtcOffset};
+
+/// Obtain the system's UTC offset.
+pub(super) fn local_offset_at(datetime: OffsetDateTime) -> Option<UtcOffset> {
+    let js_date: js_sys::Date = datetime.into();
+    let timezone_offset = (js_date.get_timezone_offset() as i32) * -60;
```

Comment:
A comment explaining why it's -60 and not 60 would probably be best for anyone not familiar with JavaScript. _I_ know it's because of the way the JS method works, but that's only due to my history as a web dev.